### PR TITLE
Wip solution rough draft

### PR DESCRIPTION
**Pull request by drakeltheryuujin**

### CODE REVIEW

**Review by jenmyers — COMMENTED** [review]
Steven had marked this one "remove .each" but that might clobber this whole lab. We might not need it.

**Comment by sgharms** [comment]
@drakeltheryuujin @jenmyers I want to see this ported to using the techniques that have been learned thus far: `while` and `Array`. So the tests can probably stay, but the README and the solution code should use only the programming techniques taught thus far. Consult the AT board for PAC1-3 to see what's "on the table" to use.

File: README.md [hunk 1]
```diff
@@ -6,11 +6,18 @@
...
+[squares](https://en.wikipedia.org/wiki/Square_(algebra)) each element in an
+array of numbers and returns a new array of these squared numbers.
+
+**Hint:** There is more than one way to square numbers! Google Ruby's exponent
+operator or refer back to the earlier lesson on simple math.
 
 ## Apply the `.each` Method to Iterate Over an Array
 
-Use an iterator and implement your own logic, *don't* use any built-in array methods other than `.each` (e.g. `.collect`, `.inject`).
+To build this method, use the iterator `.each` and implement your own logic. *Do
```
Comment (sgharms) [inline README.md:18]: remove use of .each

also include don't include exhaustive list of methods not to use do something like: "Use only while an Arrays, don't use the internet to find code you don't understand yet, etc."

File: lib/square_array.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 def square_array(array)
-  # your code here
+  squared = []
+  array.each { |element| squared << element ** 2 }
```
Comment (sgharms) [inline lib/square_array.rb:3]: use `while`

File: README.md (hunk 1 above)
Comment (drakeltheryuujin) [reply README.md:18]: @sgharms had the same thought

Comment (sgharms) [reply README.md:18]: I meant let’s not include the exhaustive list of what not to use. In PAC we haven’t taught enumerables yet. So solutions and readers should t use them.

**Pull request closed without merging**